requirements-build.txt

```diff
@@ -1,16 +1,12 @@
 #
-# This file is autogenerated by pip-compile with Python 3.11
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
shouldn't we be generating things with Python 3.11 or above?
This is just a comment, I'm aware, but out of coherency?

Reply:
fixed